### PR TITLE
docs: a quick tutorial on getting started

### PR DESCRIPTION
**Pull request by pandemicsyn**
Sending along some quick incremental doc updates. Since some community folks are already looking at the EDK 🥳 

----
:books: Documentation preview :books:: https://meltano-edk--35.org.readthedocs.build/en/35/

### CODE REVIEW

**Comment by pandemicsyn** [comment]
https://meltano-edk--35.org.readthedocs.build/en/35/getting_started.html#basic-steps-to-get-started

File: docs/getting_started.md [hunk 1]
```diff
@@ -43,3 +43,146 @@ For working examples of Meltano EDK extensions, see:
...
+- `myextensionthing_invoker`: the wrapper for the `uname` command, used for when you want to invoke the wrapped command
+
+For example to run `uname -a` you could run:
+
+```shell
+$ poetry run myextensionthing_invoker -a
+Darwin MacBook-Pro.localdomain 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:19:52 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T6000 arm64
+```
+
+But you can also call interact with the extension itself:
```
Comment (pandemicsyn) [inline docs/getting_started.md:135]:
```suggestion
But you can also interact with the extension itself:
```

File: docs/getting_started.md [hunk 2]
```diff
@@ -43,3 +43,146 @@ For working examples of Meltano EDK extensions, see:
...
++++ b/myextensionthing_ext/extension.py
+@@ -57,3 +57,5 @@ class MyExtensionThing(ExtensionBase):
+             ]
+         )
+
++    def pre_invoke(self, invoke_name: str | None, *invoke_args: Any) -> None:
++        log.info("Howdy. I am a pre-invoke task being called by the extension")
+```
+
+The line `Howdy. I am a pre-invoke task being called by the extension` will now be printed to the console when you run the `myextensionthing_invoker` command:
```
Comment (pandemicsyn) [inline docs/getting_started.md:180]:
```suggestion
The line `Howdy. I am a pre-invoke task being called by the extension` will now be printed to the console everytime you run the `myextensionthing_invoker` command:
```

File: docs/getting_started.md [hunk 3]
```diff
@@ -43,3 +43,146 @@ For working examples of Meltano EDK extensions, see:
...
+
+```shell
+$ poetry run myextensionthing_invoker -a
+Howdy. I am a pre-invoke task being called by the extension
+Darwin MacBook-Pro.localdomain 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:19:52 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T6000 arm64
+```
+
+The extension base class also provides a `post_invoke` method that can be used to run tasks after the wrapped command is invoked, head over to the [EDK reference docs](https://edk.meltano.com/en/latest/classes/meltano.edk.extension.ExtensionBase.html#meltano.edk.extension.ExtensionBase) for more information on what interface consists of and what else available.
```
Comment (pandemicsyn) [inline docs/getting_started.md:188]:
```suggestion
The extension base class also provides a `post_invoke` method that can be used to run tasks after the wrapped command is invoked, head over to the [EDK reference docs](https://edk.meltano.com/en/latest/classes/meltano.edk.extension.ExtensionBase.html#meltano.edk.extension.ExtensionBase) for more information on what the interface consists of and what else available out of the box.
```

File: docs/getting_started.md [hunk 4]
```diff
@@ -43,3 +43,146 @@ For working examples of Meltano EDK extensions, see:
...
+
+The line `Howdy. I am a pre-invoke task being called by the extension` will now be printed to the console everytime you run the `myextensionthing_invoker` command:
+
+```shell
+$ poetry run myextensionthing_invoker -a
+Howdy. I am a pre-invoke task being called by the extension
+Darwin MacBook-Pro.localdomain 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:19:52 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T6000 arm64
+```
+
+The extension base class also provides a `post_invoke` method that can be used to run tasks after the wrapped command is invoked, head over to the [EDK reference docs](https://edk.meltano.com/en/latest/classes/meltano.edk.extension.ExtensionBase.html#meltano.edk.extension.ExtensionBase) for more information on what the interface consists of and what else available out of the box.
```
Comment (pandemicsyn) [inline docs/getting_started.md:188]:
```suggestion
The extension base class also provides a `post_invoke` method that can be used to run tasks after the wrapped command is invoked, head over to the [EDK reference docs](https://edk.meltano.com/en/latest/classes/meltano.edk.extension.ExtensionBase.html#meltano.edk.extension.ExtensionBase) for more information on what the interface consists of and what else is available out of the box.
```

**Review by tayloramurphy — APPROVED** [review]
No notes - looks great! 🚢

**Review by aaronsteers — APPROVED** [review]
Looks great! 

I don't see it covered here, and might be a nice follow-on to add help on getting added to `meltano.yml`.

**Pull request merged**